test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8145,6 +8157,11 @@ def expect(self, obj_type, attrib=None, id=None, op=EQ, attrop=PTL_OR,
             else:
                 operator = op
                 val = v
+            if operator not in PTL_OP_TO_STR:
+                self.logger.log(level, "Operator not supported by expect(), "
```

Comment:
please also print which operator is not supported in the error message.

Reply:
Also I feel this piece of code needs a test case.

Reply:
The problem with printing the operator is that 'operator' is just a integer into the 'range' tuple that's defined inside pbs_testlib.py. So, when i print out the operator, the log message comes out as: "Operator 2 not supported by expect(), cannot verify change in managers", which doesn't seem very useful.
I'll try writing a test case

Reply:
Thanks for explaining. Make sense to not to include it in the message then.